applications/GeoMechanicsApplication/custom_processes/calculate_incremental_displacement_process.h

```diff
@@ -0,0 +1,37 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+#pragma once
+
+#include "processes/process.h"
+
+namespace Kratos
+{
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) CalculateIncrementalDisplacementProcess : public Process
+{
+public:
+    KRATOS_CLASS_POINTER_DEFINITION(CalculateIncrementalDisplacementProcess);
+
+    CalculateIncrementalDisplacementProcess(ModelPart& rModelPart, const Parameters& rSettings);
+
+    ~CalculateIncrementalDisplacementProcess() override;
+
+    CalculateIncrementalDisplacementProcess(const CalculateIncrementalDisplacementProcess&)            = delete;
+    CalculateIncrementalDisplacementProcess& operator=(const CalculateIncrementalDisplacementProcess&) = delete;
```

Comment:
Just a question: what is the reason we do not allow to copy objects of this type?

I would suggest to follow the [Rule of Zero](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c20-if-you-can-avoid-defining-default-operations-do) as recommended by the C++ Core Guidelines, if possible. In that case, we could also get rid of explicitly defining the class's destructor.